bucket/ricochet.json

```diff
@@ -0,0 +1,18 @@
+{
+    "version": "1.1.4",
+    "description": "Anonymous peer-to-peer instant messaging.",
+    "homepage": "https://github.com/ricochet-im/ricochet",
```

Comment:
https://ricochet.im

Reply:
It will redirect to github.

Reply:
I can without problem my VMs physically running in Czechia, Romania, Germany, Switzerland, England.

Problem is on your end.

Reply:
Thanks, you are right, it's blocked in China.